src/test/java/org/olf/reshare/dcb/DcbTest.java

```diff
@@ -26,12 +29,45 @@ class DcbTest {
   void testItWorks () {
     Assertions.assertTrue(application.isRunning());
   }
-
+  
   @Test
   void testDcbRootPath () {
-    final String text = client.toBlocking().retrieve(HttpRequest.GET("/dcb"),
+    final String text = client.toBlocking().retrieve(HttpRequest.GET("/dcb").basicAuth("user", "password"),
```

Comment:
If we are moving to rest assured can you convert this test also and then remove the client from the sepc so we keep things consistent.